Userland/Libraries/LibJS/Runtime/VM.cpp

```diff
@@ -682,6 +682,50 @@ Value VM::get_new_target()
     return verify_cast<FunctionEnvironment>(*env).new_target();
 }
 
+Object* VM::get_import_meta()
```

Comment:
This Pseudo-AO is missing a spec link in its new location. `https://tc39.es/ecma262/#sec-meta-properties-runtime-semantics-evaluation` should be included in some form here above the function definition. In some form meaning, perhaps with additional context that it's only the `ImportMeta` branch.

Reply:
Oh yes my bad, I'll make sure to keep all this stuff in mind next time so I don't cause as much review burden and force pushes